rvip/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rvip)
+
+add_compile_options(-std=c++11)
+
+find_package(catkin REQUIRED COMPONENTS
+  tf
+  rospy
+  roscpp
+  roslint
+  pcl_ros
+  std_msgs
+  cv_bridge
+  sensor_msgs
+  geometry_msgs
+  image_transport
+  visualization_msgs
+)
+
+roslint_cpp()
```

Comment:
`roslint_cpp()` is always run. Shouldn't this be conditional upon `CATKIN_ENABLE_TESTING` instead?

Reply:
Updated. [See Commit](https://github.com/cardboardcode/rvip-1/commit/5f40c5a16c7589521c0a2c7b27c662937dee1c7b)